### DISCUSSION TITLE
CIV2UIA

### DISCUSSION THREAD

**Discussion by reubene** [axx0/Civ2-clone#86] [Ideas]
CIV2 UIA which I found out about here (https://forums.civfanatics.com/threads/civilization-ii-mge-user-interface-additions-civ2uia.623515/) adds some interesting QOL features to civ2.

Do you think we should incorporate these changes in our interface as we build it?  Some of them are no brainers like the scroll bars but others like the progress numbers for settlers ruin the immersion for players that are less into optomization

**Comment by axx0** [comment]
I was also thinking about this. My suggestion is this: if 99% of players make use of some feature (like scrolling or changes you made to TOT main menu) then incorporate it and no option to turn it off is needed. Hardcore purists will probably play the original game anyway.
Then if there are other features which most players will use, make an option to turn it off but it should be ON by default.
Other more controversial options should be off by default. Perhaps the settlers' progress numbers fall into this category.
The latest issue of texture filtering should also be an option.

I don't know if these options should be in the current appsettings or some other ini/json file. If you have any ideas about this go ahead.

**Comment by crhy** [comment]
For the record I have only played the CIV2UIA version for years now.  And the numbers for settlers building stuff is a huge plus.  No reason to stay true to the original as long as the gameplay is super fun like the original.  QoL features I say yes to across the board.  Gameplay changes I usually say no to, although it would be cool to toggle nukes, a few other things that are not currently togglable.
